WeatherMapDataSource_prometheus_manualtest.php

```diff
@@ -11,21 +11,27 @@
 
 
 //$prom_query = 'prometheus:http:localhost:9090:192.168.178.32:Gi0/13:ifHCInOctets:ifHCOutOctets';
+//$prom_query = 'prometheus:http:localhost:9090:sum(irate(ifHCOutOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]))*8:sum(irate(ifHCInOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]))*8';
```

Comment:
FWIW the IN should appear first, to be in line with the in_query and out_query

Reply:
Fixed,